modin/_compat/pandas_api/py36/dataframe.py

```diff
@@ -53,6 +54,30 @@ def apply(
             func, axis=axis, raw=raw, result_type=result_type, args=args, **kwds
         )
 
+    def compare(
+        self,
+        other,
+        align_axis=1,
+        keep_shape=False,
+        keep_equal=False,
+    ):
+        return self._compare(
+            other=other,
+            align_axis=align_axis,
+            keep_shape=keep_shape,
+            keep_equal=keep_equal,
+            result_names=("self", "other"),
```

Comment:
Why do we pass this parameter for the old version?

Reply:
added a comment